src/yaml/fukuda/blam-hidp-fx-chemicals.yaml

```diff
@@ -0,0 +1,50 @@
+group: blam
+name: hidp-fx-chemicals
+version: "1.0"
+subfolder: 400-industrial
+info:
+  summary: BLaM HIDP FX Chemicals
+  description: |-
+    FX Chemicals started their production in 1954, and were bought by the Gasol Corporation in december of the year 1987, they produce top-quality light ethylenes for variate industries (you can find them everywhere, in diverse forms)
+
+    Stats:
+     - Location: Landmarks
+     - Size: 8x2
+     - Bulldoze cost: 100
+     - I-D Jobs: 544
+  author: fukuda
+  website: https://community.simtropolis.com/files/file/17591-blam-hidp-fx-chemicals/
+  images:
+    - https://www.simtropolis.com/objects/screens/0017/93f1338768b381f7cccaa4b9aa5ffcf3-fx1.JPG
+    - https://www.simtropolis.com/objects/screens/0017/93f1338768b381f7cccaa4b9aa5ffcf3-fx2.JPG
+dependencies:
+  - blam:hidp-fx-chemicals-resources
+assets:
+  - assetId: blam-hidp-fx-chemicals
+    include:
+      - .*\.SC4Lot$
```

Comment:
Ah, thanks for clarifying. I wasn't really sure how it worked. I assumed that once if detects that something might be a regex - like when it includes stuff like * or $, it treats it as a regex, and otherwise just checks for a substring.